packages/jobs/lib/execution/operations/utils/errors.ts

```diff
@@ -0,0 +1,27 @@
+import { NangoError, deserializeNangoError } from '@nangohq/shared';
+import { stringifyError } from '@nangohq/utils';
+
+export function getFormattedScriptError({ err, defaultErrorType, scriptName }: { err: unknown; defaultErrorType: string; scriptName: string }) {
```

Comment:
also would something like `toNangoError(...)` or `asNangoErro()` be a more descriptive name?